package.json

```diff
@@ -4,6 +4,9 @@
   "author": "Takuya Asano <takuya.a@gmail.com>",
   "description": "JavaScript implementation of Japanese morphological analyzer",
   "main": "./dist/node/kuromoji.js",
+  "browser": {
+    "./dist/node/loader/NodeDictionaryLoader.js": "./dist/node/loader/BrowserDictionaryLoader.js"
```

Comment:
I got it!
`b.require` is needed for such that modules:
1. require kuromoji.js (in node.js manner)
2. are browserified (for working on browser)